### ISSUE TITLE
[Accessibility][Cross-browser] Multiple selection Dropdown: Cannot remove selected item

### ISSUE THREAD

**Issue by kamilakorzec** [microsoft/fluent-ui-react#1138]
### Steps
1. Select one or more items in multiple selection dropdown
2. Focus one of the items
3. Try to remove the focused item

### Expected Result
'X' button is focusable and actionable OR pressing 'Enter' removes the focused item

### Actual Result
Cannot focus 'X' button on item pill or remove the item with keyboard keys

### Version
0.25.1

### Testcase
[Fork, update, and replace this pen to show the bug]:
Reproducible in sample component on GitHub page

**Comment by jurokapsiar** [comment]
This works as designed. User can use arrow keys to select the pill and press delete or backspace to remove. Closing the issue, please reopen if you think this does not work correctly.
